src/org/exist/xquery/ErrorCodes.java

```diff
@@ -213,6 +213,7 @@
     public static final ErrorCode FOJS0003 = new W3CErrorCode("FOJS0003", "JSON duplicate keys.");
 
     public static final ErrorCode FOUT1170 = new W3CErrorCode("FOUT1170", "Invalid $href argument to fn:unparsed-text() (etc.)");
+    public static final ErrorCode FOQM0002 = new W3CErrorCode("FOQM0002", "Module URI not found.");
     
```

Comment:
I think we are missing `FOQM0001`, `err:FOQM0003`, `err:FOQM0005` and `err:FOQM0006`.
Although `err:FOQM0006` is unlikely in eXist-db.